generator/bindata.go

```diff
@@ -729,34 +729,34 @@ func AssetNames() []string {
 // _bindata is a table, holding each asset generator, mapped to its name.
 var _bindata = map[string]func() (*asset, error){
```

Comment:
The changes in this file are just the result of a `gofmt -s -w -l .`, no changes in logic. If needed they could be rolled back before merging